apps/ledger-live-desktop/src/renderer/components/DeviceAction/rendering.tsx

```diff
@@ -49,9 +49,18 @@ import { swapDefaultTrack } from "~/renderer/screens/exchange/Swap2/utils/index"
 import { context } from "~/renderer/drawers/Provider";
 import { track } from "~/renderer/analytics/segment";
 import { DrawerFooter } from "~/renderer/screens/exchange/Swap2/Form/DrawerFooter";
-import { Flex, Icons, Text as TextV3, Log, ProgressLoader, BoxedIcon } from "@ledgerhq/react-ui";
+import {
+  Button as ButtonV3,
+  Flex,
+  Icons,
+  Text as TextV3,
+  Log,
+  ProgressLoader,
+  BoxedIcon,
+} from "@ledgerhq/react-ui";
```

Comment:
Would it be too hard to actually adapt the usages of the old Text to the ne w one for the whole file instead of importing both? I see only 9 usages and you'd just need to introduce the capitalize variant on the component.